.github/workflows/ci.yml

```
@@ -698,3 +698,77 @@ jobs:
             echo "Build is invalid"
             exit 1
           fi
+  prerelease_approve:
```

We use kebab case for every action in this file.
```suggestion
  prerelease-approve:
```